include/sycl_ext_complex.hpp

```diff
@@ -256,8 +256,12 @@ template<class T> complex<T> tanh (const complex<T>&);
 #define _SYCL_EXT_CPLX_FAST_MATH
 #endif
 
+#define _SYCL_BEGIN_NAMESPACE namespace sycl {
```

Comment:
I think we maybe need to be careful with that, as we did for the other namespace, because of hipsycl where the namespace sycl will just be an alias to `cl::`so we will need to extent the base namespace and not `sycl`

Reply:
Just so we are on the same page, do you mean this?

```
#ifndef _SYCL_CPLX_NAMESPACE
#ifdef __HIPSYCL__
#define _SYCL_CPLX_NAMESPACE hipsycl::sycl::ext::cplx
#else
#define _SYCL_CPLX_NAMESPACE sycl::ext::cplx
#endif
#endif
```

So defining `_SYCL_BEGIN_NAMESPACE` to `sycl` if `__HIPSYCL__` does not exist, else `hipsycl::sycl`

Reply:
Modified the namespace to also handle hypsycl, is that what you had in mind?

Reply:
@TApplencourt